offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/product/create/CreateProductView.groovy

```diff
@@ -38,13 +38,12 @@ class CreateProductView extends HorizontalLayout{
 
     ComboBox<String> productUnitComboBox
     ComboBox<String> productCategoryComboBox
-
-    Button createProductButton
     Button abortButton
 
-    Registration registration
-
-    Label label
+    /** These variables are adapted in the {@link CopyProductView} */
```

Comment:
I am not sure if you should comment on it there since this view is not depending on the copy product view. Although I really like that you did it. Do you see my point or do you prefer to keep it?

Reply:
```suggestion
```

Reply:
I completely see your point concerning the dependency. However i think in this special case it makes sense to keep it so future developers don't have a nasty surprise when they want to change this view. 👍

Reply:
But then our design is bad. I think the parent should not know about stuff in the child class.

Reply:
It should determine the values.. and in the child class (because it is inheriting) you can point out which values you are depending on. Since you implement this class it should be straight forward to get the error.

Reply:
Hmm that's also true. I'll remove the comment